cirq-core/cirq/optimizers/two_qubit_gate_math_utils_test.py

```diff
@@ -4,7 +4,7 @@
 
 import cirq
 from cirq import value
-from cirq_google.optimizers.two_qubit_gates.math_utils import (
+from cirq.optimizers.two_qubit_gate_math_utils import (
```

Comment:
Let's rename `two_qubit_gate_math_utils.py` to `gate_tabulation_math_utils.py` for now. 

Ideally, we should split up this math_utils file and move the functions to more appropriate places; for example `unitary_entanglement_fidelity` should go to `cirq-core/cirq/qis/measures.py`;  KAK related methods should go to `cirq-core/cirq/linalg/decompositions.py`; `random_qubit_unitary` and `vector_kron` should also go somewhere in `cirq/linalg`. 

If you are willing to do the split in this PR, go for it! Or else, I'm also fine with creating a new issue to track this migration and mark it as P0 so that we split up the math_utils in a separate PR before the next release.

Reply:
Done.